### ISSUE TITLE
HTTP/2 stream 1 was not closed cleanly before end of the underlying stream

### ISSUE THREAD

**Issue by AydenLii** [AbdBarho/stable-diffusion-webui-docker#249]
Has this issue been opened before?

 It is not in the [FAQ](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ), I checked.
 It is not in the [issues](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues?q=), I searched.
Describe the bug

Does not work properly and prompts
’HTTP/2 stream 1 was not closed cleanly before end of the underlying stream‘

Which UI

auto

Hardware / Software
Tested on two machines

OS: Ubuntu 20
OS version: 20.04.5 LTS
WSL version (if applicable): 2
Docker Version: 20.10.21
Docker compose version: v2.12.2
Repo version: from master
RAM: 186GB
GPU/VRAM: NVIDIA® T4 GPU 16GB*2
Steps to Reproduce

docker compose --profile auto up --build

Additional context
Change the Http protocol to 1.1 in the local Git, still can't solve the problem

**Comment by AbdBarho** [comment]
@AydenLii can you provide the logs for the problem?

**Comment by AydenLii** [comment]
**Now there are new problems. Whether sudo or not, the errors are the same**

 root@Z:~/stable-diffusion-webui-docker-master# sudo docker compose --profile auto up --build                                                                                   [+] Building 125.2s (32/37)                                                                                                                                                                                     
 => [internal] load build definition from Dockerfile                                                                                                                                                       0.0s  => => transferring dockerfile: 32B                                                                                                                                                                        0.0s  => [internal] load .dockerignore                                                                                                                                                                          0.0s 
 => => transferring context: 2B                                                                                                                                                                            0.0s
 => resolve image config for docker.io/docker/dockerfile:1                                                                                                                                                 1.6s
 => CACHED docker-image://docker.io/docker/dockerfile:1@sha256:9ba7531bd80fb0a858632727cf7a112fbfd19b17e94c4e84ced81e24ef1a0dbc                                                                            0.0s
 => [internal] load .dockerignore                                                                                                                                                                          0.0s
 => [internal] load build definition from Dockerfile                                                                                                                                                       0.0s
 => [internal] load metadata for docker.io/library/alpine:3                                                                                                                                                1.3s
 => [internal] load metadata for docker.io/library/python:3.10-slim                                                                                                                                        1.1s
 => [internal] load metadata for docker.io/alpine/git:2.36.2                                                                                                                                               1.5s
 => [internal] load build context                                                                                                                                                                          0.0s
 => => transferring context: 123B                                                                                                                                                                          0.0s
 => [xformers 1/3] FROM docker.io/library/alpine:3@sha256:8914eb54f968791faf6a8638949e480fef81e697984fba772b3976835194c6d4                                                                                 0.0s
 => [stage-2  1/16] FROM docker.io/library/python:3.10-slim@sha256:030ead045da5758362ae198e9025671f22490467312dbad9af6b29a6d6bc029b                                                                        0.0s
 => [download 1/8] FROM docker.io/alpine/git:2.36.2@sha256:ec491c893597b68c92b88023827faa771772cfd5e106b76c713fa5e1c75dea84                                                                                0.0s
 => CACHED [stage-2  2/16] RUN pip install torch==1.12.1+cu116 torchvision==0.13.1+cu116 --extra-index-url https://download.pytorch.org/whl/cu116                                                          0.0s
 => CACHED [stage-2  3/16] RUN apt-get update && apt install fonts-dejavu-core rsync git jq moreutils -y && apt-get clean                                                                                  0.0s
 => CACHED [stage-2  4/16] RUN <<EOF (git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui.git...)                                                                                            0.0s
 => CACHED [xformers 2/3] RUN apk add aria2                                                                                                                                                                0.0s
 => CACHED [xformers 3/3] RUN aria2c -x 10 --dir / --out wheel.whl 'https://github.com/AbdBarho/stable-diffusion-webui-docker/releases/download/3.1.0/xformers-0.0.15.dev0+4e3631d.d20221125-cp310-cp310-  0.0s
 => CACHED [stage-2  5/16] COPY --from=xformers /wheel.whl xformers-0.0.15-cp310-cp310-linux_x86_64.whl                                                                                                    0.0s
 => CACHED [stage-2  6/16] RUN pip install xformers-0.0.15-cp310-cp310-linux_x86_64.whl && rm xformers-0.0.15-cp310-cp310-linux_x86_64.whl                                                                 0.0s
 => CACHED [download 2/8] RUN <<EOF (cat <<'EOE' > /clone.sh...)                                                                                                                                           0.0s
 => CACHED [download 3/8] RUN . /clone.sh taming-transformers https://github.com/CompVis/taming-transformers.git 24268930bf1dce879235a7fddd0b2355b84d7ea6   && rm -rf data assets **/*.ipynb               0.0s
 => CACHED [download 4/8] RUN . /clone.sh stable-diffusion https://github.com/CompVis/stable-diffusion.git 69ae4b35e0a0f6ee1af8bb9a5d0016ccb27e36dc   && rm -rf assets data/**/*.png data/**/*.jpg data/*  0.0s
 => CACHED [download 5/8] RUN . /clone.sh CodeFormer https://github.com/sczhou/CodeFormer.git c5b4593074ba6214284d6acd5f1719b6c5d739af   && rm -rf assets inputs                                           0.0s
 => CACHED [download 6/8] RUN . /clone.sh BLIP https://github.com/salesforce/BLIP.git 48211a1594f1321b00f14c9f7a5b4813144b2fb9                                                                             0.0s
 => CACHED [download 7/8] RUN . /clone.sh k-diffusion https://github.com/crowsonkb/k-diffusion.git 60e5042ca0da89c14d1dd59d73883280f8fce991                                                                0.0s
 => CACHED [download 8/8] RUN . /clone.sh clip-interrogator https://github.com/pharmapsychotic/clip-interrogator 2486589f24165c8e3b303f84e9dbbea318df83e8                                                  0.0s
 => CACHED [stage-2  7/16] COPY --from=download /git/ /stable-diffusion-webui                                                                                                                              0.0s
 => CACHED [stage-2  8/16] RUN mkdir /stable-diffusion-webui/interrogate && cp /stable-diffusion-webui/repositories/clip-interrogator/data/* /stable-diffusion-webui/interrogate                           0.0s
 => CACHED [stage-2  9/16] RUN pip install -r /stable-diffusion-webui/repositories/CodeFormer/requirements.txt                                                                                             0.0s
 => CACHED [stage-2 10/16] RUN [[ "0" == "0" ]] && : || pip install tensorflow-cpu==2.10 tensorflow-io==0.27.0 git+https://github.com/KichangKim/DeepDanbooru.git@edf73df4cdaeea2cf00e9ac08bd8a9026b7a7b2  0.0s
 => ERROR [stage-2 11/16] RUN pip install opencv-python-headless   git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379   git+https://github.com/openai/CLIP.git@d50d7  121.9s
------                                                                                                                                                                                                          
 > [stage-2 11/16] RUN pip install opencv-python-headless   git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379   git+https://github.com/openai/CLIP.git@d50d76daa670286dd6cacf3bcd80b5e4823fc8e1   pyngrok:                                                                                                                                                                                  
#0 0.243 + pip install opencv-python-headless git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379 git+https://github.com/openai/CLIP.git@d50d76daa670286dd6cacf3bcd80b5e4823fc8e1 pyngrok                                                                                                                                                                                                     
#0 0.636 Collecting git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379                                                                                                       
#0 0.637   Cloning https://github.com/TencentARC/GFPGAN.git (to revision 8d2447a2d918f8eba5a4a01463fd48e45126a379) to /tmp/pip-req-build-9nv_3gxq
#0 0.638   Running command git clone --filter=blob:none --quiet https://github.com/TencentARC/GFPGAN.git /tmp/pip-req-build-9nv_3gxq
#0 120.9   fatal: unable to access 'https://github.com/TencentARC/GFPGAN.git/': GnuTLS recv error (-110): The TLS connection was non-properly terminated.
#0 120.9   error: subprocess-exited-with-error
#0 120.9   
#0 120.9   × git clone --filter=blob:none --quiet https://github.com/TencentARC/GFPGAN.git /tmp/pip-req-build-9nv_3gxq did not run successfully.
#0 120.9   │ exit code: 128
#0 120.9   ╰─> See above for output.
#0 120.9   
#0 120.9   note: This error originates from a subprocess, and is likely not a problem with pip.
#0 120.9 error: subprocess-exited-with-error
#0 120.9 
#0 120.9 × git clone --filter=blob:none --quiet https://github.com/TencentARC/GFPGAN.git /tmp/pip-req-build-9nv_3gxq did not run successfully.
#0 120.9 │ exit code: 128
#0 120.9 ╰─> See above for output.
#0 120.9 
#0 120.9 note: This error originates from a subprocess, and is likely not a problem with pip.
#0 121.8 
#0 121.8 [notice] A new release of pip available: 22.2.2 -> 22.3.1
#0 121.8 [notice] To update, run: pip install --upgrade pip
------
failed to solve: executor failed running [/bin/bash -ceuxo pipefail pip install opencv-python-headless   git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379   git+https://github.com/openai/CLIP.git@d50d76daa670286dd6cacf3bcd80b5e4823fc8e1   pyngrok]: exit code: 1




(yolo-mask) root@Z:~/stable-diffusion-webui-docker-master#  pip install --upgrade pip
Looking in indexes: http://mirrors.cloud.aliyuncs.com/pypi/simple/
Requirement already satisfied: pip in /root/anaconda3/envs/yolo-mask/lib/python3.8/site-packages (22.3.1)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

**Comment by DevilaN** [comment]
> Follow the prompts to upgrade pip (but my version is the latest)
Nope. You are upgrading pip on your host, and NOTICE is from container created specially by docker to build your image.
Nonetheless this should have no impact at all.

Problem is strictly with cloning git repository:
> git clone --filter=blob:none --quiet https://github.com/TencentARC/GFPGAN.git /tmp/pip-req-build-9nv_3gxq did not run successfully.

Are you using proxy or try to connect from country which has some fancy limitations (like national firewall protecting users from themselves)?
